ironfist/src/cpp/analytics.h

```diff
@@ -1,3 +1,7 @@
 #include <string>
 
-void send_event(std::string category, std::string action);
+extern std::string gameAction;
+extern std::string mapAction;
+extern std::string open;
+
+void send_event(std::string category, std::string action);
```

Comment:
Make these references so that you don't need to copy the parameter to pass in it. Make these const to enforce that this method will not change the values.